### PR TITLE
Add API documentation and validator

### PR DESCRIPTION
**Pull request by PCalape**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
     const app = await NestFactory.create(AppModule);
+
+    const config = new DocumentBuilder()
+    .setTitle('Cats example')
```
Comment (arnellebalane) [inline src/main.ts]: @PCalape The indentation looks off, was this file formatted by Prettier?

**Review by arnellebalane — CHANGES_REQUESTED** [review]
@PCalape Please also add a section in the readme on how to run the project locally, including the API docs.

File: src/main.ts (hunk 1 above)
Comment (PCalape) [reply src/main.ts]: Hello @arnellebalane! Sorry i forgot to setup prettier formatting upon file saving in my ide. Applied the formatting for all thanks!

**Review by arnellebalane — APPROVED** [review]

**Comment by PCalape** [comment]
> @PCalape Please also add a section in the readme on how to run the project locally, including the API docs.

Hello @arnellebalane updated the readme file for this one. Thanks!

**Pull request merged**